src/svg-icons/Icon.js

```diff
@@ -0,0 +1,27 @@
+import React, {PropTypes} from 'react'
+import cn from 'classnames'
+import getColored from './getColored'
+import injectSheet from '../jss'
+
+const Icon = ({sheet: {classes}, className, name, color = '#000'}) => (
```

Comment:
Let's make the default color `currentColor` so it gets it from the current text color. 
This means that in many cases we won't have to pass a color.